### ISSUE TITLE
Transform H36M Dataset to COCO Style

### ISSUE THREAD

**Issue by HansMertens** [facebookresearch/VideoPose3D#67]
Hello there,
i'd like to train a model with just the 6 keypoints that are relevant for me. Therefore i trained a Mask R-CNN via Detectron on the COCO Dataset reduced to the 6 keypoints. Your Paper suggests fine-tuning on H36M afterwards. Is there a script to transform the H36M dataset to a COCO-Style Dataset so i can fine-tune it via Detectron?
Thanks for your help

**Comment by dariopavllo** [comment]
Hi,

In the paper we state:

> When finetuning the model on Human3.6M, we reinitialize the last
layer of the keypoint network, as well as the deconv layers
that regress the heatmaps to learn a new set of keypoints.

We don't train Mask R-CNN on COCO keypoints, but we actually adapt the output head to predict Human3.6M keypoints.

**Comment by HansMertens** [comment]
Thanks for your reply,
the paper says:
„For both Mask R-CNN and CPN, we start with pre- trained models on COCO [32] and fine-tune the detectors on 2D projections of Human3.6M.“
The pretrained models are trained on COCO keypoints, aren‘t they?

**Comment by HansMertens** [comment]
One more question:
would the performance of VideoPose3D suffer if it is trained on just 6 keypoints instead of 17?

**Comment by dariopavllo** [comment]
Probably, but it depends on the settings. If you train on 6 keypoints and you try to predict all 17 keypoints, then it will definitely suffer. If you predict 6 joint positions from 6 keypoints it may be better or worse, depending on the difficulty of those keypoints (in terms of occlusions, etc.)
